### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by feyisakedir**
Pull Request
the mean edition performed on the following file
-readme. md----future features addition and description
- lint file edition
- make .gitignore file root file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,122 +36,31 @@
 > Describe the tech stack and include only the relevant sections that apply to your project.
```
Comment (V-Blaze) [inline README.md:36]: - [ ] Great job using the recommended Microverse readme template. :zap: :+1: , But as stated in the project requirement, you need to customize the readme file for your specific project, like the `Key Feature, Future Features`etc.
> Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

- Please go through the readme file and personalize the content, I'll also give some suggestions. Remember to remove the placeholder comments. :+1: :smile:

File: README.md [hunk 2]
```diff
@@ -183,9 +82,9 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- [ ] **[.css]**
+- [ ] **[.html]**
+- [ ] **[.js]**
```
Comment (V-Blaze) [inline README.md:87]: - [ ] In this section, I'll suggest you remove the comment `Describe 1 - 3 features you will add to the project.` and then add three features that you will be adding to your project later. :smile: :+1:

File: .github/test.md
Comment (V-Blaze) [inline .github/test.md:1]: - [ ] According to the project requirement you are supposed to create a `test.md` file at the root directly of your project and then add it to the `.gitignore` file. Please move this file to your project root or delete this and rename the one in the project root correctly. :+1: :smile:

File: .github/.gitignore
Comment (V-Blaze) [inline .github/.gitignore:1]: - [ ] The `.gitignore` file is meant to be in the root of your project, Please I'll suggest you move this file to your project root. :sun_with_face:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @, 

# Almost there 🟡 

## NB: I noticed that you closed the first pull request and submitted a new one. According to Microverse requirements, you are not supposed to merge or close a pull request until it has been reviewed and approved by a code reviewer, Please in your subsequent projects, do not merge or close an unapproved pull request. Happy Coding! :+1: 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, and some changes from the previous reviewers, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Linters are setup correctly✔
- Used Microverse readme template. :green_apple: 

 ## Required Changes ♻️

- [ ] According to the project requirement
> Remember to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

You are supposed to make your project look professional including your pull request, I'll suggest you edit your pull request and make a good summary, You can include the task or the changes you made in this branch as the summary. :smile: 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (feyisakedir) [reply README.md]: Can I create new pull Request?

File: README.md [hunk 3]
```diff
@@ -6,35 +6,23 @@
...
-
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
```
Comment (Eternalgratis) [inline README.md:16]: - Please consider adding the setup of your project here as well

**Review by Eternalgratis — COMMENTED** [review]
# Status: Required Changes ♻️
Hi @feyisakedir , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but some are still not addressed yet.

almost-there

 ## Highlights
✅ You used the correct GitHub flow
✅ Descriptive pull request message
✅ linter error checks validated


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -6,35 +6,23 @@
...
 </div>
-
-<!-- TABLE OF CONTENTS -->
-
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
```
Comment (Eternalgratis) [inline README.md:14]: Great job on implementing some of the changes requested by the previous reviewer. However, there are a few to make some adjustments to

- Kindly add the key features in line 17 [Key Features](#key-features) of this particular project. The essence of a customized Readme is to help anyone visiting your repo to have an idea of what the project is about

**Review by Eternalgratis — COMMENTED** [review]
# Status: Required Changes ♻️
Hi @feyisakedir , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but some are still not addressed yet.

almost-there

 ## Highlights
✅ You used the correct GitHub flow
✅ Descriptive pull request message
✅ linter error checks validated


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — COMMENTED** [review]
Hi @feyisakedir , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a pull request that has already been merged

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by feyisakedir** [comment]
It's not invalid you're the first person said invalid code
I can't create new PR b/c the previous reviewer comment is necessary

**Pull request merged**